third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -1919,11 +1953,7 @@ void LoadGLShader(GLuint shaderID, const std::string &filename) {
   if (blen > 1) {
     DO_VALIDATION;
     GLchar *compiler_log = (GLchar *)malloc(blen);
-#ifdef __APPLE__
-    mapping.glGetInfoLogARB((void *)&shaderID, blen, &slen, compiler_log);
-#else
-    mapping.glGetInfoLogARB(shaderID, blen, &slen, compiler_log);
-#endif
+    mapping.glGetShaderInfoLog(shaderID, blen, &slen, compiler_log);
```

Comment:
Instead of using ARB extension, it is better to use Core profile's functionality (it gives more meaningful errors on Mac OS)